crates/sui/tests/shared_objects_tests.rs

```diff
@@ -495,7 +495,7 @@ async fn shared_object_on_gateway() {
     // the handles (or the authorities will stop).
     let configs = test_authority_configs();
     let _handles = spawn_test_authorities(gas_objects.clone(), &configs).await;
-    let clients = create_authority_aggregator(configs.validator_set());
+    let clients = test_authority_aggregator(&configs);
```

Comment:
Just a rename for consistency (trying to get all testing facilities in the same place)